mycroft/skills/msm_wrapper.py

```diff
@@ -12,29 +12,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-"""Common logic for instantiating the Mycroft Skills Manager.
 
-The Mycroft Skills Manager (MSM) does a lot of interactions with git.  The
-more skills that are installed on a device, the longer these interactions
-take.  This is especially true at boot time when MSM is instantiated
-frequently.  To improve performance, the MSM instance is cached.
 """
+NOTE: this is dead code! do not use!
```

Comment:
Is this module ever imported now? Consider a warning/error log on import if not

Reply:
cant remove because we are drop in replacement and stuff in the wild might expect it

we do not install msm or use this file at all, but someone else might....

a log is a good idea

Reply:
Yeah, I think a warning on import lets people know it's a bad idea but doesn't break.. Then if/when Mycroft refactors it away it's easy for us to drop without any surprises

Reply:
`DeprecationWarning` "The OpenVoiceOS team regards this as dead code. It's available for backwards compatibility with upstream projects, and may be removed in a future update."